ArduPlane/quadplane.cpp

```diff
@@ -741,7 +742,7 @@ bool QuadPlane::setup(void)
     }
 
     AP_Param::load_object_from_eeprom(attitude_control, attitude_control->var_info);
-    pos_control = new AC_PosControl(*ahrs_view, inertial_nav, *motors, *attitude_control);
+    pos_control = new AC_PosControl_TS(*ahrs_view, inertial_nav, *motors, *attitude_control);
```

Comment:
I actually think this might be better where the AC_PosControl_TS always clears the I terms removing the need to pass to bool. We then use `AC_PosControl` on everything except vectored tailsitters where we use this  new method. Should be a much smaller change set and a little clearer what is going on and stops taillsiter changes taking out normal quadplanes.

Reply:
would you do that by casting pos_control to AC_PosControl_TS?